docs/DevGuide/Amr.md

```diff
@@ -0,0 +1,367 @@
+corresponding to the listed tags; and
+amr::projectors::CopyFromCreatorOrLeaveAsIs that will either copy the
+item from the parent (or children, asserting that all children agree)
+of a newly created element, or leave the item unmodified for an
+existing element for the items corresponding to the listed tags.
+
+The list of projectors is specified in
+`metavariables::amr::projectors` in the executable, and the projectors
+are evaluated in the order in which they are specified.
+
+### Enabling AMR for an executable {#dev_guide_amr_metavariables}
+
+In order to enable AMR for an executable, the following changes need
+to be made in the metavariables:
+
+- The addition of a struct named `amr` such as the following example:
+```
+struct amr : tt::ConformsTo<::amr::protocols::AmrMetavariables> {
+    using element_array = dg_element_array;
+    using projectors = tmpl::list<::amr::projectors::DefaultInitialize<
+        domain::Tags::InitialExtents<Dim>,
+        domain::Tags::InitialRefinementLevels<Dim>,
+        evolution::dg::Tags::Quadrature>>;
+  };
+```
+
+  where `element_array` specifies the array component on which AMR
+  will operate and `projectors` is a type list of amr::projectors
+  that govern how the items in the DataBox for the `element_array` are
+  updated after refinement.
+
+- The addition of amr::Component to the `component_list` of the metavariables
+
+- The addition (or modification) of the following `tmpl::pair`s in
+  `factory_creation::factory_classes`:
+```
+        tmpl::pair<amr::Criterion,
+                   tmpl::list<LIST_OF_CRITERIA>,
+        tmpl::pair<
+            PhaseChange,
+            tmpl::list<
+                PhaseControl::VisitAndReturn<
+                    Parallel::Phase::EvaluateAmrCriteria>,
+                PhaseControl::VisitAndReturn<Parallel::Phase::AdjustDomain>,
+                PhaseControl::VisitAndReturn<Parallel::Phase::CheckDomain>>,
+```
+where `LIST_OF_CRITERIA` should be a list of amr::Criteria.
+
+- The addition of the following item in the list of PhaseActions for
+  the component specified in `amr::element_array`:
+
+```
+          Parallel::PhaseActions<Parallel::Phase::CheckDomain,
+                                 tmpl::list<::amr::Actions::SendAmrDiagnostics,
+                                            Parallel::Actions::TerminatePhase>>,
+```
+
+- The possible addition of `PhaseControl::Actions::ExecutePhaseChange`
+  to the action list for the appropriate Phase in the PhaseAction of
+  the `amr::element_array`
+
+- The addition of the mutator `amr::Initialization:Initialize` to the
+  list of mutators in `Initialization::Actions::InitializeItems` in
+  the Parallel::PhaseActions list for Parallel::Phase::Initialization.
+
+- The appropriate includes for all of the above.
+
+### Controlling AMR {#dev_guide_amr_input_options}
+
+There are two places in the input file that control how and when AMR happens.
+
+The how is controlled by the option group `Amr`.  Here you list the
```

Comment:
Could you add quotes around "how"? Thought this was a typo when I first looked at it